.eslintrc

```
@@ -397,6 +397,9 @@
     "wrap-iife": [
       "error",
       "any"
-    ]
+    ],
+    // note you must disable the base rule as it can report incorrect errors
```

Always start comment with a capital letter and punctuation.
```suggestion
    // The following rule must be disabled so that we can enable "@typescript-eslint/dot-notation".
```